sdk/search/tests.yml

```diff
@@ -5,6 +5,7 @@ jobs:
     parameters:
       ServiceDirectory: search
       Timeout: 120
+      MaxParallel: 2
       EnvVars:
         AZURE_TEST_MODE: LIVE
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```

Comment:
Can these variables be deleted now that we have the `test-resources.json` file?

Reply:
Deleted, Search doesn't use them in testing anyways.